.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 notifications:
   email: false
-sudo: false
+sudo: required
```

Comment:
Travis runs much more slowly with `sudo` required, is this absolutely necessary?